R/expectation.R

```diff
@@ -98,6 +98,24 @@ as_label_flt  <- function(quo) {
     paste0("`", quo_lab, "`")
 }
 
+# check for data frames provided as vars ----
+
+check_expect_data_pipe <- function(first_arg) {
+  if (as_label(first_arg) == ".")
+    stop(
+      "Your `vars` argument is a `.`. ",
+      "Did you accidentally use a `%>%` pipe into an expectation?\n",
```

Comment:
Maybe another new line here? When the console is cramped the single new line looks slightly off

![image](https://user-images.githubusercontent.com/61998181/141672097-b771e103-c940-4a06-a5b3-f844ef0d0156.png)

Reply:
Done!